docs/src/overview/yaml/platform-yaml-tags.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Platform.sh YAML tags
+weight: 0
+description: A description of custom YAML tags available for Platform.sh files.
+---
+
+In addition to the [basic functions you should be familiar with](./what-is-yaml.md), YAML allows for special tags.
+Platform.sh accepts certain custom tags to facilitate working with configuration files.
+
+These tags work with Platform.sh configuration files, but may not elsewhere.
+
+## Include
+
+Use the `!include` tag to embed external files within a given YAML file.
+
+The tag requires two properties:
+
+| Property | Type     | Possible values               | Description |
+| -------- | -------- | ----------------------------- |
+| `type`   | `string` | `string`, `binary`, or `yaml` | See the descriptions of [strings](#string), [binaries](#binary), and [YAML](#yaml). Defaults to `yaml`. |
+| `path`   | `string` |                               | The path to the file to include, relative to the directory the YAML file is in. |
+
+### `string`
+
+Use `string` to include an external file inline in the YAML file as if entered as a multi-line string.
+
+For example, if you have a build hook like the following:
+
+```yaml {location=".platform.app.yaml"}
+hooks:
+    build: |
+        set -e
+        cp a.txt b.txt
+```
+
+You could create a file for the script:
+
+```text {location="build.sh"}
+set -e
+cp a.txt b.txt
+```
+
+And replace the hook with an include tag for an identical result:
+
+```yaml {location=".platform.app.yaml"}
+hooks:
+    build: !include
+        type: string
+        path: build.sh
+```
+
+This helps you break longer configuration like build scripts out into a separate file for easier maintenance.
+
+### `binary`
+
+Use `binary` to include an external binary file inline in the YAML file.
+The file is base64 encoded.
+
+For example, you could include a `favicon.ico` file in the same folder as your app configuration.
+Then you can include it as follows:
+
+```yaml {location=".platform.app.yaml"}
+properties:
+    favicon: !include
+        type: binary
+        path: favicon.ico
+```
+
+### `yaml`
+
+Use `yaml` to include an external YAML file inline as if entered directly.
+Because `yaml` is the default you can use it without specifying the properties.
```

Comment:
Hmm. That loses the point of the two lines. They're not really redundant at all, but I can try rewording to make it clearer.

Reply:
This comment confused me because it's not showing the two lines you're describing as being redundant. Once I figured it out, I realized it was because one was left in by mistake. Next time, you can highlight multiple lines at once for a comment: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Reply:
Apologies for that, indeed should have made it clearer.